gui.js

```diff
@@ -1480,6 +1481,41 @@ IDE_Morph.prototype.createCorralBar = function () {
         this.corralBar.left() + padding + newbutton.width() + padding
     );
     this.corralBar.add(paintbutton);
+
+    cambutton = new PushButtonMorph(
+        this,
+        "newCamSprite",
+        new SymbolMorph("camera", 15)
+    );
+
+    cambutton.corner = 12;
+    cambutton.color = colors[0];
+    cambutton.highlightColor = colors[1];
+    cambutton.pressColor = colors[2];
+    cambutton.labelMinExtent = new Point(36, 18);
+    cambutton.padding = 0;
+    cambutton.labelShadowOffset = new Point(-1, -1);
+    cambutton.labelShadowColor = colors[1];
+    cambutton.labelColor = this.buttonLabelColor;
+    cambutton.contrast = this.buttonContrast;
+    cambutton.drawNew();
+    cambutton.hint = "take a camera snapshot and\nimport it as a new sprite";
+    cambutton.fixLayout();
+    cambutton.setCenter(this.corralBar.center());
+    cambutton.setLeft(
+        this.corralBar.left() + padding + newbutton.width() + padding + paintbutton.width() + padding
+    );
+
+    if (location.protocol === 'http:') {
+        cambutton.hint = 'Due to browser security policies, you need to\n' +
```

Comment:
Should we maybe name this 'camera requires https' or something and stuff it in a translation file? (Would make it easier to translate, and reduces a bit of duplication below).

Reply:
this is an excellent, idea, @cycomachead !
@bromagosa , I can take care of this after I pull it, so don't worry about it. I'd like to pull this when we reach consensus about "file:" protocol  and Michael's other remarks. What do you think?

Reply:
Cool! Thank you, Jens! :)

But we do not have an English translation file, do we?

Reply:
Haha, in fact, we do have an English translation file for this exact purpose :-)

Reply:
Oh wow, I didn't know we did! I can't find it anywhere, what's it called?

Reply:
It's inside ```locale.js```. But nevermind, I'll take care of it!